README.md

```diff
@@ -36,13 +36,44 @@ Commands:
   help [command]                   display help for command
 ```
 
-For `OpenAI` config:
+`registry` config:
 
 ```bash
 Options:
-  --openaiKey <value>      recommend alternative packages via ChatGPT
-  --openaiModel <value>    ChatGPT model (choices: "gpt-3.5-turbo", "gpt-4", "gpt-4-turbo", "gpt-4o-mini", "gpt-4o")
-  --openaiBaseURL <value>  override the default base URL for the API
+  --registry <value>        specify registry URL
+```
+
+`OpenAI` config:
+
+```bash
+Options:
+  --openaiKey <value>       recommend alternative packages via ChatGPT
+  --openaiModel <value>     ChatGPT model (choices: "gpt-3.5-turbo", "gpt-4", "gpt-4-turbo", "gpt-4o-mini", "gpt-4o")
+  --openaiBaseURL <value>   override the default base URL for the API
+```
+
+For `current`:
+
+```bash
+Options:
+  --ignore <value>          ignore specific packages
+  --failfast                exit the program if it has been deprecated
+```
+
+For `global`:
+
+```bash
+Options:
+  -m, --manger <value>      check specified package manger (choices: "npm", "yarn", "pnpm")
+  --ignore <value>          ignore specific packages
```

Comment:
I think it would be nice if the description is clear about how to ignore more then one package.